MobiusCore/Source/Mobius.swift

```diff
@@ -57,26 +59,23 @@ public extension Mobius {
         private let effectHandler: AnyConnectable<Effect, Event>
         private let initiator: Initiator<Model, Effect>
         private let eventSource: AnyEventSource<Event>
-        private let eventQueue: DispatchQueue
-        private let effectQueue: DispatchQueue
         private let logger: AnyMobiusLogger<Model, Event, Effect>
+        private let eventFilter: ConsumerFilter<Event>
```

Comment:
This would require event transformers to be a nominal type rather than a function type, though. Currently “event filters” are in fact composable by adding more than one, although restricted to not changing the event type (that could easily be added)

Reply:
Mmm. https://github.com/pointfreeco/swift-overture#pipe is maybe one option. Again, probably a discussion best left until later.

Reply:
I’m leaning towards `eventConsumerTransformer`. It’s bulky but accurate, and it’s an advanced use case.